backend/api/src/resources/messages-response.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
What (if any) benefits are there to using json type versus enum type?

Reply:
Some of the benefits:

- When using **json type**, you can reassign or change the member values
- JavaScript, does not support **enums**, but TypeScript helps us access them
- When using **enum type** and you plan to reassign or change the enum member values, enums are type-safe and, therefore, would return compile errors on reassignment
- When you want to record dynamic values, **enums** are best suited for finite items, and the general idea behind it was to help create a user-defined constants system
- **Enums** cannot be used as variables; doing so would return errors
- With **enums**, you can create constants that you can easily relate to, making constants more legible